### ISSUE TITLE
How to update Jasmine version

### ISSUE THREAD

**Issue by zivni** [wallabyjs/public#539]
It seems the wallaby is currently using an older version of Jasmine. Is there a way to update the version to a newer one?

I'm using vscode on windows. current wallaby version 1.0.11

**Comment by ArtemGovorov** [comment]
We are planning to add the latest Jasmine version support soon (and make it to be the default version). In a meantime, if you have the latest version of Jasmine somewhere in your project, you may do this in your wallaby config:

```javascript
module.exports = function () {
  return {
    files: [
      ...
    ],

    tests: [
      ...
    ],

    testFramework: { type: 'jasmine', path: 'path/to/jasmine/in/your/project' }
  };
};
```

Let me know if it helps.

**Comment by ArtemGovorov** [comment]
BTW, what new features of Jasmine did you want to use, so that you need the latest version?

**Comment by ArtemGovorov** [comment]
*Update*: I have added `jasmine 2.4.1` support to the latest core v1.0.210. The core will be updated automatically within a few minutes after your next wallaby.js (re)start. You may also force the core update by opening your `C:\Users\user\.vscode\extensions\Wallaby-vscode\` folder, deleting the `wallaby` subfolder in there and restarting your VS Code.

You may use the new version of jasmine like this:
```javascript
module.exports = function () {
  return {
    files: [
      ...
    ],

    tests: [
      ...
    ],

    testFramework: 'jasmine@2.4.1'
  };
};
```

**Comment by zivni** [comment]
Wow, thanks for the quick update!

I was just trying to use `toHaveBeenCalledTimes` when I noticed that wallaby uses an older version. Now it works.

Good to know there is a way to set the path to to the testing framework. I should add it to the documentation (I couldn't find it if it exists). It would be good to add it to [Supported technologies page](http://wallabyjs.com/docs/integration/overview.html), as this is what google found when I searched.
